### ISSUE TITLE
Not making the form disappear or saying message sent

### ISSUE THREAD

**Issue by emmamargaret** [kocakmhmt/SMTP-PHP-Contact-Form#3]
Hello Mehmet,
Firstly thank you so much for making this available - I searched for ages for a form like this which I was able to set up - being a beginner at this stuff.
I have just installed the form on a test site, on the Contact tab - http://emmagallagher.co.uk/pathfinder/contacttest.php It all seem sto be working I am getting most of the test emails. Strangely not all.
But my main ifficulty is that when I press send the form does not disappear and there is no - Thanks for sending your message. I had too edit the form as there were too many boxes to fill.
Any thought gratefully received and many thanks in advance
All the best,
Emma

Here is the code

HTML:<div id="contactcontainer">
    <form class="form" id="contact_form" name="contact_form" method="post" action="javascript:void(0);">
    <p class="name">
        <label for="name">Name<font color="red">_</font></label>
    <input type="text" name="name" id="name" placeholder="Your Name_"  required/>
    </p>
    <p class="email">
        <label for="email">Email<font color="red">_</font></label>
    <input type="text" name="email" id="email" placeholder="mail@example.com_" required />
    </p>
    </p>
         <img src="SMTP-PHP-Contact-Form-master/inc/captcha.php" />
    <label for="guvenlikKodu"><font color="red">_</font></label>
        <input type="text" name="guvenlikKodu" id="guvenlikKodu" required />
        <p class="text">
         <label for="text">Message<font color="red">_</font></label>
    <textarea id="text" name="text" id="text"  required/></textarea>
    </p>
    <p class="submit">
    <input type="submit" onclick="kontrolet()" value="Send" />
    </p>
    </form>
    </div>

SCRIPT          function gostergizle()
            {
            $("div#container").show(100); 
            }
            function sendmessage(){ 
            var veriler = $('#contact_form').serialize(); 
            $.ajax({ 
            type: "POST", 
            url: "SMTP-PHP-Contact-Form-master/inc/send.php", 
            data: veriler, 
            success:function(cevap){ 
            $("#send").html(""+cevap); 
            $("div#container").hide(100); 
            } 
            })}; 
            function formReset(id) {
            $('#'+id).each(function(){
            this.reset();
            });
            }
            function kontrolet(){
        var name=$('#name').val();
            name=jQuery.trim(name);
        var email=$('#email').val();
            email=jQuery.trim(email);
        var mess=$('#text').val();
            mess=jQuery.trim(mess);
        var tel=$('#phone').val();
            tel=jQuery.trim(tel);
        var guvenlikKodu=$('#guvenlikKodu').val();
            guvenlikKodu=jQuery.trim(guvenlikKodu);        

```
        if(name==''){
        alert("Please check your name");
        }   
        else if(email==''){
            alert("Please check your E-Mail");
            }

        else if (guvenlikKodu==''){
            alert("Security code is empty!");
            }
        else if(mess==''){
            alert("Please check your Message");
            }
        else { sendmessage() } 
            }
```

SEND.PHP 
<?php
session_start();
if ( isset($_POST['guvenlikKodu']) && $_POST['guvenlikKodu'] ){
 $guvenlikKontrol = false;
 if ( $_POST['guvenlikKodu'] == $_SESSION['guvenlikKodu'] ){
 $guvenlikKontrol = true;
 }
  if ( $guvenlikKontrol ){
$text=$_POST['text'];
$name=$_POST['name'];
$email=$_POST['email'];

require("class.phpmailer.php");
$mail = new PHPMailer();
$mail->IsSMTP();
$mail->Host     = "smtp.sendgrid.net."; // SMTP servers
$mail->SMTPAuth = true;     // turn on SMTP authentication
$mail->Username = "computerfitness.net";  // SMTP username
$mail->Password = "password"; // SMTP password
$mail->From     = "contact@computerfitness.net"; // it must be a match with SMTP username
$mail->Fromname = "computerfitness.net"; // from name
$mail->AddAddress("emmagallagher10@gmail.com"); // SMTP username , Name Surname
$mail->Subject  =  $_POST[ 'Message from computerfitness'];
$content = "<h2>You have a message from computerfitness.net</h2>  <p><b>Name:</b>$name</p> <p><b>E-Mail:</b>$email</p> <p><b>Message:</b>$text</p>" ;
$mail->MsgHTML($content);
if(!$mail->Send())
{
   echo "<center>Error! Its wrong!</center>";
   echo "Mailer Error: " . $mail->ErrorInfo;
    echo "<center><p><input type='submit' onclick='gostergizle();' value='Back' /></p></center>";
   exit;
}
echo "<center>Thank you! Your message has reached us! <p><input type='submit' onclick='gostergizle();' value='Back' /></p></center>";
 } else {
 echo "<center>Please check Security Code! <p><input type='submit' onclick='gostergizle();' value='Back' /></p></center>";
 }
}
?>

**Comment by emmamargaret** [comment]
Have converted the files into one document so you can read it  - sorry about the last message!
Many thanks
emma
[GITHUB.docx](https://github.com/kocakmhmt/SMTP-PHP-Contact-Form/files/158957/GITHUB.docx)

**Comment by kocakmhmt** [comment]
Hi Emma,
Thanks for issue.

Firstly I saw 2 loaded jQuery on your website. Could you try only one? 
and you have to add this codes into head tags for hide-unhide form : 

```

    <script type="text/javascript"> 
            function gostergizle()
            {
            $("div#container").show(100); 
            }
            function sendmessage(){ 
            var veriler = $('#contact_form').serialize(); 
            $.ajax({ 
            type: "POST", 
            url: "inc/send.php", 
            data: veriler, 
            success:function(cevap){ 
            $("#send").html(""+cevap); 
            $("div#container").hide(100); 
            } 
            })}; 
            function formReset(id) {
            $('#'+id).each(function(){
            this.reset();
            });
            }
            function kontrolet(){
        var name=$('#name').val();
            name=jQuery.trim(name);
        var email=$('#email').val();
            email=jQuery.trim(email);
        var mess=$('#text').val();
            mess=jQuery.trim(mess);
        var tel=$('#phone').val();
            tel=jQuery.trim(tel);
        var guvenlikKodu=$('#guvenlikKodu').val();
            guvenlikKodu=jQuery.trim(guvenlikKodu);        

            if(name==''){
            alert("Please check your name");
            }   
            else if(email==''){
                alert("Please check your E-Mail");
                }
            else if (tel==''){
                alert("Please check your Phone");
                }
            else if (guvenlikKodu==''){
                alert("Security code is empty!");
                }
            else if(mess==''){
                alert("Please check your Message");
                }
            else { sendmessage() } 
                }
</script>
```

Please try this ways and if it doesn't work I will help you again.

Thanks for using

**Comment by emmamargaret** [comment]
Hi Mehmet,
I've removed the extra jQuery code and added the script to the head tags -
but it's still not hiding the form - sorry.
Thanks for your help
Emma

**Comment by kocakmhmt** [comment]
You have to add code as I wrote. 
Begining of the code, you wrote like that `<script>` but it should be like that `<script type="text/javascript">`

Please fix and try.

Please don't hesitate to ask any question.
Thanks

**Comment by emmamargaret** [comment]
I did that and refreshe and sent the message - the message came through but
the form still din't disappear!
Thank you so much.

**Comment by kocakmhmt** [comment]
Please remove these codes : 
`<script type="text/javascript" src="http://code.jquery.com/jquery-latest.min.js"></script>
`
and
`<script type='text/javascript' src='http://ajax.googleapis.com/ajax/libs/jquery/1.3/jquery.min.js'></script>
`

**Comment by emmamargaret** [comment]
Hi I have removed and tried again, no change - am getting the messages
though
Emma

**Comment by emmamargaret** [comment]
Hi Mehmet,
Would it be possible to keep the form in place and just add a "message
sent" under the Send button once it has been sent? Would that be easy to
add?
Many thanks again
Emma

**Comment by kocakmhmt** [comment]
Hi,
you should change to this code: 
`$("div#container").show(100);` to `$("div#contactcontainer").show(100);`
and `$("div#container").hide(100);` to  `$("div#contactcontainer").hide(100);`

**Comment by emmamargaret** [comment]
Hi Mehmet,
I'm so sorry - I should have seen that. So I changed the div id of the
whole form back to id="container" and the form disappears and the message
come throgh super fast now - great!
The only thing that doesn't happen is that there is no text to say that the
message was sent successfully. Perhaps it is white? Do you have any ideas?
Thank you so much,
Emma
